cmd/oci/oci.go

```diff
@@ -37,6 +45,9 @@ var OCICmd = &cobra.Command{
 	bsf oci <environment name> 
 	bsf oci <environment name> --platform <platform>
```

Comment:
Let's change this from "environemnt name" to "artifact" maybe?

Reply:
artifact name?